plot_explorer.py

```diff
@@ -409,6 +432,55 @@ def editColorBy(self, domain_kind, apply=False):
         if apply:
             self.applyChanges()
 
+    def editColorMap(self, colormap_name, property_type, apply=False):
+        self.model.activeView.colormaps[property_type] = colormap_name
+        self.plotIm.updateColorMap(colormap_name, property_type)
+        self.colorDialog.updateColorMaps()
+        if apply:
+            self.applyChanges()
+
+    def editColorBarMin(self, min_val, property_type, apply=False):
```

Comment:
Matplotlib documentation talks about "colorbar" as a single word, so I'd name this `editColorbarMin` (with similar changes elsewhere)

Reply:
mmm good call. Will update